guides/image-sources/prov-create-host-pxe.iuml

```diff
@@ -0,0 +1,17 @@
+User -> Foreman : create host action
+Foreman -> Proxy : request a free IP address
+Proxy -> DHCP : search for a free IP in DHCP
+DHCP -> Proxy : return the free IP
+Proxy -> Foreman : pass the free IP
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : render boot-loader configuration files
+end
+Proxy -> TFTP : deploy boot-loader configuration files\nto ""/var/lib/tftpboot""
+Foreman -> Proxy : command to download installer kernel\nand initial RAM disk
+Proxy -> TFTP : download kernel and initial RAM disk
```

Comment:
```suggestion
Proxy -> TFTP : deploy installer kernel\nand initial RAM disk
```
@stejskalleos Could we simplify these two steps into one like this^^?

Reply:
In my opinion, it's as good as it is now.